pkg/build/options.go

```diff
@@ -36,19 +45,19 @@ func (o *Options) Validate() error {
 
 func (o *Options) implementationFromStrategy() error {
 	switch o.Strategy {
```

Comment:
nit: feels like we should do switch `bazelStrategy(o.Strategy)` or make the `o.Strategy` type a `bazelStrategy` and compare to the constants directly but /shrug